tests/DCPS/Compiler/optional/optional.idl

```diff
@@ -1,8 +1,24 @@
 module optional {
 
-  @topic
-  struct OptionalMembers {
-    @optional long value;
+  typedef sequence<int32> seq_int_t;
+
+  struct TestStruct {
+    octet octect_field;
   };
 
+  @appendable
+  struct OptionalMembers {
+    boolean bool_field;
```

Comment:
should `boolean` be optional?

Reply:
This was originally added to debug some issues with a struct with both optional and non-optional members, but can be made optional now.